docker-compose.yml

```diff
@@ -60,6 +64,8 @@ services:
     env_file:
       - .dockerenv
       - .secrets
+    external_links:
+      - italia_docs_italia_it_nginx:local.docs.italia.it
```

Comment:
questo a cosa serve?

Reply:
Questo serve a Premailer per poter recuperare i CSS. L'url che gli viene passato è composto a partire dal `Site` model, e quindi è necessario far sì che `local.docs.italia.it` venga risolto con l'ip di nginx.
Un'alternativa a `external_links` è quella di creare un alias basato sulla direttiva `networks` di `docker-compose`, ma l'intervento è più invasivo perché deve essere aggiunta esplicitamente a tutti i services (vedere https://docs.docker.com/compose/compose-file/#aliases).

Reply:
ok, visto che comunque è solo per sviluppo locale lo possiamo lasciare. mi preoccupava il dominio martellato. Le variabvili di ambiente non possiamo usarle qui, giusto?

Reply:
Good point, verifico.

Reply:
Fixed in https://github.com/italia/docs.italia.it/pull/287/commits/b3490bc97f880a323d22bad67038ae074e5fbfef

Non sono variabili d'ambiente però: ho preferito usare la direttiva ARG del Dockerfile, mi sembrava più appropriata (e non aggiunge entry all'env).

Reply:
non capisco il collegamento
 :thinking: 

Gli arg per la build docker è bene che rimangano nel dockerfile perché il docker compose non viene usato in deployment

Reply:
Sorry, avevo capito male io: pensavo volessi usare docker-compose per iniettare le versioni nel Dockerfile. Ripristino quanto fatto a com'era prima.

Tornando alle env var: docker-compose supporta l'uso dei variabili d'ambiente che stanno all'interno della shell in cui viene lanciato (https://docs.docker.com/compose/compose-file/#variable-substitution)

Reply:
forse possiamo usare anche quelle prese da `env` perồ giusto?

Reply:
Dentro il docker-compose.yml possiamo usare solo le env var specificate tramite file .env (https://docs.docker.com/compose/env-file/). La direttiva `env_file` (che noi usiamo con i valori `.dockerenv` e `.secrets` serve invece ad iniettare env var nei container a runtime).

Attualmente non disponiamo di un file `.env`, posso crearlo in questa PR.

Reply:
Vedi https://github.com/italia/docs.italia.it/pull/287/commits/92e4f72271e2fb26d551ed7b526c1da3e24dae90